### PR TITLE
Add tabbed_view with one tab listing all workspaces to workspace root

### PR DESCRIPTION
**Pull request by maethu**
This PR bases on #3824, please merge this one first.

Currently we have only one tab... I know this is no useful, but it will be replaces really soon.

### CODE REVIEW

**Comment by maethu** [comment]
@jone updated.

**Comment by jone** [comment]
@maethu screenshot? 😏

File: opengever/workspace/browser/tabs.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+         'groupable': False,
+         'width': 30},
+
+        {'column': 'reference',
+         'column_title': _(u'label_reference', default=u'Reference Number')},
+
+        {'column': 'Title',
+         'column_title': _(u'label_title', default=u'Title'),
+         'sort_index': 'sortable_title',
+         'transform': linked},
```
Comment (jone) [inline opengever/workspace/browser/tabs.py]: Could you add default widths to the columns? Would improve the initial impression; we are therefore trying to do that 😉 x

**Comment by maethu** [comment]
@jone There you are:
screen shot 2017-12-21 at 10 52 22

**Comment by jone** [comment]
@maethu what actions are available? Do we need checkbox / actions at this point? 🤔

File: opengever/workspace/browser/tabs.py (hunk 1 above)
Comment (maethu) [reply opengever/workspace/browser/tabs.py]: @jone With a default width per column:
screen shot 2017-12-21 at 10 57 22

**Comment by maethu** [comment]
@jone Just the default ones, like, move, zip export...

**Comment by maethu** [comment]
@jone I removed the actions and the checkbox-column

**Comment by maethu** [comment]
@jone I also updated the changelog 🙈

File: docs/HISTORY.txt [hunk 2]
```diff
@@ -11,6 +11,10 @@ Changelog
 - Change msg2mime transform to use msgconvert executable from $PATH instead of shipping our own wrapper script. [lgraf]
 - Add partial reindex optimization for trashing and untrashing objects. [elioschmutz]
 - Fix notification counter, count only badge notifications. [phgross]
+- OGIP 17: Implement sequence_number and NameFromTitle behavior for Workspace. [mathias.leimgruber]
+- OGIP 17: Add tabbedviews for opengever.workspace types. [mathias.leimgruber]
+- Enable edit border for committee members on meeting view. [phgross]
```
Comment (jone) [inline docs/HISTORY.txt]: ✂️

**Pull request merged**